docker-compose-test.yml

```diff
@@ -27,23 +27,22 @@ services:
       - "6379:6379"
 
   categorisation-localstack:
-    image: localstack/localstack:0.11.2
+    image: localstack/localstack:latest
     ports:
-      - "4567-4584:4567-4584"
+      - "4566-4584:4566-4584"
     networks:
       - hmpps
     container_name: categorisation-localstack
     restart: unless-stopped
     environment:
       - SERVICES=${SERVICES-sqs}
       - DEBUG=${DEBUG- }
-      - DATA_DIR=${DATA_DIR- }
-      - PORT_WEB_UI=${PORT_WEB_UI-8088}
-      - LAMBDA_EXECUTOR=${LAMBDA_EXECUTOR- }
+      - PERSISTENCE=${DATA_DIR- }
       - KINESIS_ERROR_PROBABILITY=${KINESIS_ERROR_PROBABILITY- }
-      - DEFAULT_REGION=eu-west-2
-    volumes:
-      - './.localstack:/tmp/localstack'
+      - AWS_DEFAULT_REGION=eu-west-2
+      - LOCALSTACK_HOST=categorisation-localstack
+#    volumes:
+#      - './.localstack:/tmp/localstack'
```

Comment:
not sure if we need this? it kept a bunch of files for what seemed like debug purposes, but no issue with it being ephemeral as best i can tell